### DISCUSSION TITLE
This command must be called in a Kobweb application module.

### DISCUSSION THREAD

**Discussion by clarkshu1016** [varabyte/kobweb#397] [General]
when i try to use command : kobweb run , it showed This command must be called in a Kobweb application module.

what should I  do to fix this issue?

**Comment by bitspittle** [comment]
Did you run `kobweb run` under the `site` folder?

**Reply by clarkshu1016** [reply]
i see i made a mistake  I run kobweb not inside site folder ,thanks for explaination

**Reply by bitspittle** [reply]
You're welcome.

FWIW the `kobweb` binary could have probably done a better job helping you out. We opened https://github.com/varabyte/kobweb-cli/issues/6 to potentially improve this in the future.
